src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -59855,5 +59855,32 @@ interface I1<T>
                 Diagnostic(ErrorCode.ERR_ClassDoesntImplementInterface, "I1<int>").WithArguments("I1<T>.P1", "I1<int>").WithLocation(4, 9)
                 );
         }
+
+        [Fact, WorkItem(41481, "https://github.com/dotnet/roslyn/issues/41481")]
+        public void IncompletePropertyImplementationSyntax_01()
```

Comment:
>IncompletePropertyImplementationSyntax_01 [](start = 20, length = 41)

I guess the tests immediately above already cover this scenario with only difference that the syntax for the property here is incomplete.